src/stactools/jrc_gsw/stac.py

```diff
@@ -80,9 +81,10 @@ def create_item(
 
     agg_hrefs = {}
     for agg_type in agg_types:
-        agg_hrefs[
-            agg_type
-        ] = f"{source}/Aggregated/{DOWNLOAD_VERSION}/{agg_type}/tiles/{agg_type}-{tile_id}.tif"  # noqa
+        agg_hrefs[agg_type] = path.join(
+            source,
+            f"Aggregated/{DOWNLOAD_VERSION}/{agg_type}/tiles/{agg_type}-{tile_id}.tif",
+        )  # noqa
```

Comment:
Should we follow this pattern to be truly platform agnostic?

```python
agg_hrefs[agg_type] = path.join(
    source,
    "Aggregated",
    DOWNLOAD_VERSION,
    agg_type,
    "tiles",
    f"{agg_type}-{tile_id}.tif",
)
```

Reply:
Sure, done